.github/workflows/build.yml

```diff
@@ -409,6 +411,8 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node }}
+      - name: Set up Bun
+        uses: oven-sh/setup-bun@v1
```

Comment:
m: I'd prefer running bun tests separately because I don't think we gain a lot from running it in the Node matrix (i.e. 7 times)